### DISCUSSION TITLE
Most played albums shouldn't be based only on tracks playcount sum

### DISCUSSION THREAD

**Discussion by rombat** [navidrome/navidrome#2711] [General]
### Version

0.50.2

### Current Behavior

Actually, this issue has already been mentioned and detailed [here](https://github.com/navidrome/navidrome/issues/1032), but it was closed due to inactivity.

Each time I go to "Most played albums" page, I'm kind of annoyed by the fact that "most played" is only measured by the sum of album tracks playcount, and not the actual ratio of tracks playcount/tracks count, which imho should be the correct way to determine how much an album has been played.

For instance, I've recently listened **just once** to an album with 357 tracks (a huge game soundtrack), and it appeared in my "most played albums", which feels wrong to me.

### Expected Behavior

My feelings are detailed [here](https://github.com/navidrome/navidrome/issues/1032#issuecomment-1186093206), but here's what I would like:

the "playcount" of an album as a whole should be the ratio of **total album tracks playcount/album tracks count**.

**Comment by deluan** [comment]
Hey, thanks for using Navidrome.

This is not really a bug, it is working as intended. This is how "Most Played" is implemented in the original Subsonic and also in statistics sites like Last.fm.

I understand this is one of those situations that "one size does NOT fit all", and that means that changing this behaviour would break it for other class of users (me included).

From the issue #1032 you linked, you can see there's not really a consensus on how to solve this.

As this is not really a "bug", I'll convert it to a discussion (so it is never closed by the stale bot) and other people can join and give their opinion.

**Comment by dweymouth** [comment]
I think one way of "fixing" it could be to introduce a new Navidrome config flag, something like ND_NORMALIZEALBUMPLAYS, which is off by default, but when On, determines play count for albums by dividing track play counts by total tracks in the album as suggested.

**Comment by vvdveen** [comment]
I also think a configuration option to change this behavior would be nice. I come from Kodi, where _Album playcount is determined as the rounded average of the playcounts of all the songs on the album_. I mostly play my music per album and try to listen to those that I haven't heard frequently yet. See also an old [kodi discussion](https://forum.kodi.tv/showthread.php?tid=355032&pid=2955604).

**Comment by vvdveen** [comment]
I submitted a [pull request](https://github.com/navidrome/navidrome/pull/2803) to add this feature.
